### PR TITLE
remove stats plugin even if not on body

### PR DESCRIPTION
**Pull request by ChristopherChudzicki**
Fixes https://github.com/unconed/threestrap/issues/24.

### CODE REVIEW

File: src/extra/stats.js [hunk 1]
```diff
@@ -16,8 +16,7 @@ Bootstrap.registerPlugin("stats", {
   },
 
   uninstall: function (three) {
-    document.body.removeChild(this.stats.domElement);
-
+    this.stats.domElement.remove()
```
Comment (ChristopherChudzicki) [inline src/extra/stats.js:19]: The stats plugin is added as a child of Threestrap's element, which is not necessarily the body element.

**Comment by ChristopherChudzicki** [comment]
@sritchie I'd suggest reviewing/merging #25 first to get a more reasonable diff here.

File: src/extra/stats.js (hunk 1 above)
Comment (sritchie) [reply src/extra/stats.js:19]: ah, that is why I was hitting this! makes sense.

**Comment by sritchie** [comment]
Gah, I did this in the wrong order... sorry about that. This PR looks great, merge away!

**Comment by ChristopherChudzicki** [comment]
> Gah, I did this in the wrong order
Nah, I think you did it right... The merge conflict was inevitable since we're currently committing the minified built code

> This PR looks great, merge away!

I can't 😦 Not a collaborator here 😭.

I'm also going to need your help releasing the new version on NPM. Here's a PR with the updated version number and changelog https://github.com/unconed/threestrap/pull/27

**Pull request merged**